security/jwt/src/main/java/io/helidon/security/jwt/Jwt.java

```diff
@@ -966,23 +982,43 @@ public Errors validate(String issuer, String audience) {
      *                 issuer claim mandatory
      * @param audience validates that this JWT was issued for this audience. Setting this to non-null value and with
      *                 any non-null value in the Set will make audience claim mandatory
+     * @param checkAudience whether audience claim check is configured as mandatory. Validation will fail when {@code true}
+     *                      and audience claim is null
```

Comment:
Please correct me if I am wrong, but this is not what this code does. If I am not mistaken, if set to `false`, it does not add the validation at all (which is fine by this statement), but how about the statement about failing validation in case of null audience? Isn't it actually null if set to optional? Also there is this null check for audience which will not add this check at all :-)

Reply:
Forgot to fix this javadoc after I changed logic of the fix. Should be OK now.